lts-13.3.nix

```diff
@@ -450,7 +450,7 @@ hackage:
       "genvalidity-aeson".revision = (((hackage."genvalidity-aeson")."0.2.0.2").revisions)."ff30febbacd45805ba7d18fefec75b19aef59d2a66d2260efbb85a1f72349bac";
       "gloss-algorithms".revision = (((hackage."gloss-algorithms")."1.13.0.1").revisions)."8529bb5ff781b02df65ffe27b3891d897a34ad19311b262b4ad682bcc6ac1f90";
       "fgl".revision = (((hackage."fgl")."5.7.0.1").revisions)."92f2e8769ae12232727d7d8b773f9aac924f3c9324935690a5096efc6289fe35";
-      "ghc-heap".revision = (((hackage."ghc-heap")."8.6.3").revisions).default;
```

Comment:
I don't think we want to do this here. We'd rather want to set that to null in the `pkgs-overlay`.
```
 pkgSet = haskell.mkNewPkgSet {
    inherit pkgs;
    pkg-def = stackage.${stack-pkgs.resolver};
    pkg-def-overlays = [
      stack-pkgs.overlay
      { ghc-heap = null; }
    ];
```
I would think that should work.